src/client/components/detail-chart/detail-chart.component.js

```diff
@@ -12,106 +12,82 @@ import {
-        height={400}
-      >
-        <LineChart data={filterByMaterialId} margin={{ right: 80 }}>
-          <XAxis
-            stroke="#DDE0E3"
-            dataKey={formatXAxis}
-            name="Time"
-            height={50}
-            domain={['dataMin', 'dataMax']}
-            tick={{
-              fontFamily: 'Roboto',
-              textAlign: 'right',
-              fontSize: 8,
-              fill: '#A3A3A3',
-            }}
-          />
-          <YAxis
-            stroke="#DDE0E3"
-            type="number"
-            domain={[boundary.minimum - 3, boundary.maximum + 3]}
-            tick={{
-              transform: 'translate(-20, 0)',
-              fontFamily: 'Open Sans',
-              fontSize: 10,
-              fill: '#666666',
-            }}
-            fill="none"
-          />
-          <Tooltip />
-          <ReferenceLine
-            y={boundary.optimum}
-            stroke="#6F9C67"
-            strokeWidth={2}
-          />
-          <ReferenceLine
-            y={boundary.maximum}
-            stroke="#9C5256"
-            strokeWidth={2}
-          />
-          <ReferenceLine
-            y={boundary.minimum}
-            stroke="#9C5256"
-            strokeWidth={2}
-          />
-          <Line
-            type="monotone"
-            name={description.temperature}
-            unit={unit.temperature}
-            dataKey="reading_value"
-            stroke="#000000"
-            fill="#000000"
-            strokeWidth={2}
-          />
-          <Brush
-            className="range-bar"
-            startIndex={0}
-            endIndex={10}
-            stroke="#73ABD7"
-            height={30}
-            travellerWidth={15}
-            gap={5}
-          />
-        </LineChart>
-      </ResponsiveContainer>
-    </div>
+    <ResponsiveContainer
+      className="detail-chart-container"
+      width="100%"
+      height={400}
+    >
+      <LineChart data={data} margin={{ right: 80 }}>
+        <XAxis
+          stroke="#DDE0E3"
+          dataKey={formatXAxis}
+          name="Time"
+          height={50}
+          domain={['dataMin', 'dataMax']}
+          tick={{
+            fontFamily: 'Roboto',
+            textAlign: 'right',
+            fontSize: 8,
+            fill: '#A3A3A3',
+          }}
+        />
+        <YAxis
+          stroke="#DDE0E3"
+          type="number"
+          domain={[boundary.minimum - 3, boundary.maximum + 3]}
+          tick={{
+            transform: 'translate(-20, 0)',
+            fontFamily: 'Open Sans',
+            fontSize: 10,
+            fill: '#666666',
+          }}
+          fill="none"
+        />
+        <Tooltip />
+        <ReferenceLine y={boundary.optimum} stroke="#6F9C67" strokeWidth={2} />
+        <ReferenceLine y={boundary.maximum} stroke="#9C5256" strokeWidth={2} />
+        <ReferenceLine y={boundary.minimum} stroke="#9C5256" strokeWidth={2} />
+        <ReferenceArea
+          y1={boundary.minimum + 2}
```

Comment:
those numbers should be calculated based on values, -2 and +2 look good with temperature, and humidity, and maybe PH, but not with others